docker-compose.yml

```diff
@@ -1,21 +1,42 @@
 version: '3.5'
 
 services:
-  web:
-    container_name: hackathonmentors
-    build: .
+  django:
+    container_name: api
+    build: ./api
     command: python hackathonmentors/manage.py runserver 0.0.0.0:8000
     volumes:
-      - .:/build
+      - ./api:/build
     ports:
       - "8000:8000"
     environment:
       DJANGO_SECRET_KEY: /var/run/secrets/django_secret
       EMAIL_SERVICE_NAME: mailchimp
       EMAIL_NEWLETTER_LIST_ID: d574cb485f
+      DATABASE_HOST: db
     secrets:
       - django_secret
 
+  web:
+    container_name: web
+    image: node:15.8.0
```

Comment:
i would do node 14, odd numbers are unstable and very bleeding edge

Reply:
Gotcha, this is exactly the kind of help I needed -- I don't know much about nodejs ecosystem. 
Any specific minor version preference?